src/test/java/org/mule/extension/file/integration/FileListTestCase.java

```diff
@@ -77,13 +77,21 @@ public void listRecursiveWithSizeCheckWithFileStillBeingWritten() throws Excepti
   }
 
   @Test
-  public void listWithoutReadPermission() throws Exception {
```

Comment:
This test is fine, but I would still keep a test where you want to explicitly list a directory to which you have no permitions and you get the error ACCESS_DENIED